__tests__/app.test.js

```diff
@@ -129,6 +129,83 @@ describe("Using app.js to run the database of NC-games", () => {
         });
     });
   });
-  //Head describe DELETE later!
+  describe("6. PATCH/api/reviews/:reviews_id", () => {
+    test("201: response with a board game object with a updated vote", () => {
+      const updateVote = { votes: 2 };
+      const review_id = 2;
+      return request(app)
+        .post(`/api/reviews/${review_id}`)
+        .send(updateVote)
+        .expect(201)
```

Comment:
200 instead of 201  - 201 is generally reserved for new posts or new creation of data, this is just an edit of data

Reply:
All 200 status now I thought because it was a update/ patch of data it would be 201 however as you stated above 201 = creation